ui/lib/core/addon/components/form-field.js

```diff
@@ -58,7 +58,10 @@ export default class FormFieldComponent extends Component {
     return this.args.disabled || false;
   }
   get showHelpText() {
-    return this.args.showHelpText || true;
```

Comment:
this was my bad 🤦🏻‍♀️

Reply:
And yeah, ideally we wouldn't return a true by default on a param, but this is an old pattern and if we choose to modify should be done in a cleanup ticket.